### DISCUSSION TITLE
Add an additional parameter for body text to __write_org_subitem() + others

### DISCUSSION THREAD

**Discussion by metbril** [novoid/Memacs#125] [Q&A] [answered]
I am looking for a way to add body text to a header. Like this:

```
** <2023-11-27 Mon 11:15> This is the header
This is the body text.
  :PROPERTIES:
  :END:
```

I have been looking around the Memacs tools and haven't been able to find a way to do this.

Adding `\nThis is body text` to the output format does not work. This is added to the header literally.

Is this even possible at all with the current converters?

**Comment by novoid** [comment]
Well, at least by convention, body text is added after the PROPERTIES drawer.

My personal approach was to generate memacs output files without much of the content but more containing a link to the original data source with all the details.

I did this for multiple reasons. First, to speed up Memacs output generation. Second, to minimize (redundant) data. And lastly because if you don't include everything from the original source, you'll end up looking up the original data anyways. So why duplicating their content?

If a body should be added, `__write_org_subitem()` needs to be extended with an optional parameter for a multi-line string containing the body. Same holds true for all other functions where `__write_org_subitem()` is used including the modules that should use it. If you know how to write basic Python, this should not be a hard problem.

However, I understand that there might be some use-cases where a body would be handy.

**Comment by metbril** [comment]
My specific use case is that my bank statements are provided as a csv file with a column "description" that is quite lengthy. I was thinking of storing that as the body instead of as a property.

**Reply by novoid** [reply, marked as answer]
I don't want to impose something onto you but I don't think that there are any length rules for PROPERTIES. At least I've added even some multi-line properties although I'm not sure if this causes any issue.

**Comment by novoid** [comment]
I still see the potential in the idea and so I added https://github.com/novoid/Memacs/issues/126 to my long list so that it is not forgotten.
